L1Trigger/L1THGCal/python/hgcalTriggerPrimitiveDigiProducer_cfi.py

```diff
@@ -65,7 +65,11 @@
 
 C3d_parValues = cms.PSet( dR_multicluster = cms.double(0.01), # dR in normalized plane used to clusterize C2d
                           minPt_multicluster = cms.double(0.5), # minimum pt of the multicluster (GeV)
-                          calibSF_multicluster = cms.double(1.084)
+                          calibSF_multicluster = cms.double(1.084),
+                          type_multicluster = cms.string('DBSCANC3d'), # 'dRC3d' for the cone algorithm 
+                          dist_dbscan_multicluster = cms.double(0.03),
```

Comment:
If I understand correctly this parameter plays the same role as `dR_multicluster`. If this is the case one single parameter could be used.